installer/templates/phx_single/lib/app_name/application.ex

```diff
@@ -13,7 +13,9 @@ defmodule <%= @app_module %>.Application do
       # Start the Ecto repository
       <%= @app_module %>.Repo,<% end %>
       # Start the PubSub system
-      {Phoenix.PubSub, name: <%= @app_module %>.PubSub},
+      {Phoenix.PubSub, name: <%= @app_module %>.PubSub},<%= if @mailer do %>
+      # Start Finch
+      {Finch, name: <%= @app_module %>.Finch}<% end %>
```

Comment:
@chrismccord We have missing comma `,` character at the end of this line which causes a compile error:

```elixir
Compiling 14 files (.ex)

== Compilation error in file lib/example/application.ex ==
** (SyntaxError) lib/example/application.ex:20:7: syntax error before: 'ExampleWeb'
    |
 20 |       ExampleWeb.Endpoint
    |       ^
    (elixir 1.15.0-dev) lib/kernel/parallel_compiler.ex:346: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
```

Reply:
good catch!